### PR TITLE
Optional information for local installation

### LINKED ISSUE #15772: Add guidence on using `bundle config --local path vendor/bundle`
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll

### What part(s) of the article would you like to see updated?

Include an optional step with `bundle config --local path vendor/bundle`

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bkmgit**
### Why:

Closes [issue link](https://github.com/github/docs/issues/15772)

### What's being changed:

By using `bundle config --local path vendor/bundle` one can do a local in folder installation that may be easier to manage and not conflict with other installations.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by WATCHARA24MONTHON — COMMENTED** [review]
Thanks!

File: content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ Before you can use Jekyll to create a {% data variables.product.prodname_pages %
 
    The correct version Jekyll will be installed as a dependency of the `github-pages` gem.
 1. Save and close the Gemfile.
+1. Optionally run `bundle config --local path vendor/bundle` so that a local folder is made for installing the gems.
```
Comment (Redapple0077) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md:92]: Always winner. Thank you so much all world

File: content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md [hunk 2]
```diff
@@ -89,6 +89,7 @@ Before you can use Jekyll to create a {% data variables.product.prodname_pages %
...
 1. From the command line, run `bundle install`.
```
Comment (Redapple0077) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md:93]: Always....

File: content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md [hunk 3]
```diff
@@ -89,6 +89,7 @@ Before you can use Jekyll to create a {% data variables.product.prodname_pages %
...
 1. Optionally, make any necessary edits to the `_config.yml` file. This is required for relative paths when the repository is hosted in a subdirectory.  For more information, see "[Splitting a subfolder out into a new repository](/github/getting-started-with-github/using-git/splitting-a-subfolder-out-into-a-new-repository)."
    ```yml
```
Comment (Redapple0077) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md:95]: Big winner

**Comment by ramyaparimi** [comment]
@bkmgit 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by ethanpalm** [comment]
👋🏼  Hi @bkmgit, thanks for creating this PR. We don't document optional workarounds like this for GitHub features, so I'm going to close this PR. We appreciate your time and interest in improving the GitHub docs! Thank you for looking for ways to help out other devs 😄

**Pull request closed without merging**